cluster-operator/src/test/java/io/strimzi/operator/cluster/model/KafkaBridgeConfigurationBuilderTest.java

```diff
@@ -592,4 +599,56 @@ public void testHttp() {
                 "http.producer.enabled=true"
         ));
     }
+
+    @ParallelTest
+    public void testWithStrimziMetricsReporter() {
+        StrimziMetricsReporterModel model = new StrimziMetricsReporterModel(
+                new KafkaBridgeSpecBuilder()
+                        .withNewStrimziMetricsReporterConfig()
+                                .withNewValues()
+                                    .withAllowList("kafka_producer_producer_metrics.*,kafka_producer_kafka_metrics_count_count")
+                                .endValues()
+                        .endStrimziMetricsReporterConfig()
+                        .build(), List.of(".*"));
+
+        String configuration = new KafkaBridgeConfigurationBuilder(Reconciliation.DUMMY_RECONCILIATION, BRIDGE_CLUSTER, BRIDGE_BOOTSTRAP_SERVERS)
+                .withStrimziMetricsReporter(model)
+                .build();
+
+        assertThat(configuration, isEquivalent(
+                "bridge.id=my-bridge",
+                "kafka.bootstrap.servers=my-cluster-kafka-bootstrap:9092",
+                "bridge.metrics=" + StrimziMetricsReporter.TYPE_STRIMZI_METRICS_REPORTER,
+                "kafka.metric.reporters=" + StrimziMetricsReporterConfig.KAFKA_CLASS,
+                "kafka." + StrimziMetricsReporterConfig.ALLOW_LIST + "=kafka_producer_producer_metrics.*,kafka_producer_kafka_metrics_count_count",
+                "kafka." + StrimziMetricsReporterConfig.LISTENER_ENABLE + "=false",
+                "kafka.security.protocol=PLAINTEXT"
+        ));
+    }
+
+    @ParallelTest
+    public void testWithPrometheusJmxExporter() {
```

Comment:
Do we have some test for the legacy metrics?

Reply:
There was no test. Do we want to add one now that is deprecated?

Reply:
Ok, will add one.